### ISSUE TITLE
[Accessibility-Color C3.2] Contrast problem between the article body and its view number with the  background

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3251]
**Environment:** Tribe

**Step of reproduction:**

- Create an article an post it in the stream
- Access to the stream page

**Current behavior:**

There is a contrast problem between the body of the article and the view number of it with the background

- **Text color:** #707070
- **Background color:** #F2F2F2
- **Contrast ratio:** 4.4

**Expected behavior:**

use the text color #F2F5F2 to have **4.5** contrast ratio at least

![Image](https://github.com/user-attachments/assets/13e8391b-d225-41d9-8a94-f36c4e21eafe)

**Comment by mbensalem** [comment]
Should be fixed by another issue
